uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/BAO/AbsencePeriod.php

```diff
@@ -208,19 +208,15 @@ public static function getMaxWeight() {
   }
 
   /**
-   * This uses PHP's date_parse to check if the given date is valid.
+   * Checks if the date is valid and in the Y-m-d format
    *
-   * The date will be valid it no error or warning is found while parsing it.
-   *
-   * @param $date The date to be checked
+   * @param string $date The date to be validated
    *
    * @return bool
    */
   private static function isValidDate($date)
   {
-    $parsed = date_parse($date);
-
-    return $parsed['warning_count'] == 0 && $parsed['error_count'] == 0;
+    return CRM_HRLeaveAndAbsences_Validator_Date::isValid($date);
```

Comment:
You comment actually made me realize we don't need the isValidDate method anymore. I could just use CRM_HRLeaveAndAbsences_Validator_Date::isValid instead